### DISCUSSION TITLE
Feature Request: Support oauth2 for webhook channel

### DISCUSSION THREAD

**Discussion by joeblubaugh** [grafana/grafana#61170] [Alerting]
The Prometheus alertmanager supports oauth2 configuration for webhooks. https://prometheus.io/docs/alerting/latest/configuration/#oauth2

It would be good for Grafana Alerting to support Oauth2 as well, perhaps using the same name for the various configuration fields.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82815. Feel free to continue the discussion around this there. Thank you!
